pyproject.toml

```diff
@@ -309,6 +310,10 @@ ignore_messages = "^.*(Unknown target name|Undefined substitution referenced)[^`
 ignore_errors = true
 scripts_are_modules = true
 strict = false
+modules = ["mne"]
+# Avoid the conflict between mne/__init__.py and mne/__init__.pyi by ignoring the former
+exclude = '^mne/(beamformer|channels|commands|datasets|decoding|export|forward|gui|html_templates|inverse_sparse|io|minimum_norm|preprocessing|report|simulation|source_space|stats|time_frequency|utils|viz)?/?__init__\.py$'
+additional_dependencies = ["numpy==1.26.2"]
```

Comment:
Why is this needed?

Reply:
That was cruft from the pre-commit hook and it's gone now